pkg/pmem-device-manager/pmd-ndctl.go

```diff
@@ -90,11 +90,13 @@ func (pmem *pmemNdctl) CreateDevice(name string, size uint64, nsmode string) err
 		glog.V(4).Infof("Device with name: %s already exists, refuse to create another", name)
 		return fmt.Errorf("CreateDevice: Failed: namespace with that name exists")
 	}
-	// align up by 1 GB, also compensate for libndctl giving us 1 GB less than we ask
-	var align uint64 = 1024 * 1024 * 1024
-	size /= align
-	size += 2
-	size *= align
+	// Increase to minimum size supported by libndctl.
+	const minsize uint64 = 2 * 1024 * 1024 * 1024
+	if size < minsize {
+		size = minsize
+	}
+	// use align by 1 GB in creation request
+	const align uint64 = 1024 * 1024 * 1024
```

Comment:
Just curious: do we have documentation about these limitations (minimum size and alignment)? As it stands here, these are magic numbers with no explanation.